packages/twenty-server/src/engine/api/graphql/workspace-query-runner/factories/query-runner-args.factory.ts

```diff
@@ -75,18 +75,20 @@ export class QueryRunnerArgsFactory {
       case ResolverArgsType.FindDuplicates:
         return {
```

Comment:
const args = args as FindDuplicatesResolverArgs; to avoid redundant type assertions?